### ISSUE TITLE
selectOptions, disabled

### ISSUE THREAD

**Issue by bazineta** [NYTimes/backbone.stickit#183]
Other than pre-rendering, is there a method by which to indicate that a given select option should be present in the select list, but disabled, i.e.,

```
<option value="foo" disabled>Foo</option>
```

**Comment by sirianni** [comment]
Any updates on this?  @yamsellem fix of adding a disabled attribute alongside 'value' and 'label' seems like a reasonable approach.  Any interest in pulling this upstream?

**Comment by sirianni** [comment]
Pre-rendering does not seem to be a viable workaround.  When the model is changed, stickit's default update handler blows away the select options and recreates them without the `disabled` attribute:
```
...
$el.find('*').remove();
...
```

**Comment by yamsellem** [comment]
@sirianni Are you saying that the fix (merged in 0.9.0) won't work in some cases?

**Comment by sirianni** [comment]
@yamsellem No, I'm saying that pre-rendering the select options (as suggested by @delambo as a workaround) does not work properly in most cases.  I have reviewed the code for your fix, but have not tried it yet in my environment (I didn't realize that 0.9.0 had been released).  I'll give it a shot soon.
